cspell.json

```diff
@@ -30,6 +24,7 @@
   ],
   "ignorePaths": [
     "/cspell.words.txt",
+    "/cspell.json",
```

Comment:
Do we need to ignore the cspell.json? 🤔 That's not something I've seen in their docs

Reply:
Interestingly enough it seems so: (see previous commit run)
https://github.com/goalgorilla/open_social/actions/runs/5423242290/jobs/9860910946